PItoADHReadOnly/Program.cs

```diff
@@ -58,27 +58,27 @@ public static async Task<bool> MainAsync(bool test = false)
             try
             {
                 // Get time indices for the last day
-                var currentTime = DateTime.UtcNow;
+                DateTime currentTime = DateTime.UtcNow;
 
                 // Format indices with "O" to use ISO 8601 standard timestamps
-                var startIndex = (currentTime - TimeSpan.FromDays(1)).ToString("O");
-                var endIndex = currentTime.ToString("O");
+                string startIndex = (currentTime - TimeSpan.FromDays(1)).ToString("O");
+                string endIndex = currentTime.ToString("O");
 
                 Console.WriteLine("Step 2. Retrieve stream");
-                var stream = await metadataService.GetStreamAsync(streamId).ConfigureAwait(false);
+                SdsStream stream = await metadataService.GetStreamAsync(streamId).ConfigureAwait(false);
                 Console.WriteLine($"Found stream: {stream.Id}");
 
                 Console.WriteLine("Step 3. Retrieve Window events");
-                var windowEvents = await dataService.GetWindowValuesAsync<PItoADHEvent>(streamId, startIndex, endIndex).ConfigureAwait(false);
+                System.Collections.Generic.IEnumerable<PItoADHEvent> windowEvents = await dataService.GetWindowValuesAsync<PItoADHEvent>(streamId, startIndex, endIndex).ConfigureAwait(false);
```

Comment:
I would just add a using for this to avoid having to include this specification throughout this file